x-pack/plugins/security_solution/common/api/detection_engine/model/alerts/8.11.0/index.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ALERT_WORKFLOW_ASSIGNEE_IDS } from '@kbn/rule-data-utils';
+import type { AlertWithCommonFields800 } from '@kbn/rule-registry-plugin/common/schemas/8.0.0';
+import type {
+  Ancestor890,
+  BaseFields890,
+  EqlBuildingBlockFields890,
+  EqlShellFields890,
+  NewTermsFields890,
+} from '../8.9.0';
+
+/* DO NOT MODIFY THIS SCHEMA TO ADD NEW FIELDS. These types represent the alerts that shipped in 8.11.0.
+Any changes to these types should be bug fixes so the types more accurately represent the alerts from 8.11.0.
+If you are adding new fields for a new release of Kibana, create a new sibling folder to this one
+for the version to be released and add the field(s) to the schema in that folder.
+Then, update `../index.ts` to import from the new folder that has the latest schemas, add the
+new schemas to the union of all alert schemas, and re-export the new schemas as the `*Latest` schemas.
+*/
+
+export type { Ancestor890 as Ancestor8110 };
+
+export interface BaseFields8110 extends BaseFields890 {
+  [ALERT_WORKFLOW_ASSIGNEE_IDS]: string[];
```

Comment:
as ALERT_WORKFLOW_ASSIGNEE_IDS defined as not required
```
  [ALERT_WORKFLOW_ASSIGNEE_IDS]: {
    type: 'keyword',
    array: true,
    required: false,
  },
```

should it be added here as optional property?

Reply:
good question, I mimicked same behaviour we have for `workflow_tags` that we introduced in 8.9 https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/api/detection_engine/model/alerts/8.9.0/index.ts#L29. It is also not required. @dplumlee do you remember why we skipped making tags optional property?

Reply:
Made this field optional, @dplumlee let me know if there is a reason to keep it required here.

Reply:
I don't recall if there was a specific reason for not making the tags optional here, I believe since it was a new field, our use case just wanted to always count on it being there. I don't see any issue as keeping aligned with the field maps definition as @vitaliidm points out though, I probably shouldn't have listed ours as optional in that file.